### ISSUE TITLE
Cannot create PRs

### ISSUE THREAD

**Issue by airon90** [LonamiWebs/Stringlate#161]
Hi, I'm using latest F-Droid version of Stringlate on Android 7.1.2 and I cannot create PRs, meanwhile I can create bug reports.

I tried to give ```adb logcat -s io.github.lonamiwebs.stringlate``` in order to give help but only
```
--------- beginning of crash
--------- beginning of system
--------- beginning of main
````
is shown. What should I do to send a log? ^^'

**Comment by gsantner** [comment]
Which version of Stringlate are you using? Maybe too, which repo doesnt work?

**Comment by julianaito** [comment]
Hi, same issue here, it forks without issues but then no pull requests or even a stringlate branch is added to the forked repo. 

**Details** : 

- **OS** LineageOS 14.1 (Android 7.1.2), latest update
- **Phone Model** Motorola XT1676 armeabi-v7a (Moto G5)
- **Repos impacted** Tried several, it won't work (KeepassDX, SimpleMobileTools/*)
-  **Logcat** [logcat_stringlate.log](https://github.com/LonamiWebs/Stringlate/files/1802265/logcat_stringlate.log)


i've let a KeepassDX repo as forked by Stringlate @ https://github.com/julianaito/KeePassDX

Hope this helps :)

**Comment by gsantner** [comment]
Logcat guide update: Maybe take a look at `https://gsantner.net/android-contribution-guide/?packageid=io.github.lonamiwebs.stringlate#logcat` (has already filled out the right package id)

**Comment by Lonami** [comment]
Fixed through #186. Sorry for the huge delay.
